src/views/Sale/components/SelfBidList/index.spec.tsx

```diff
@@ -24,7 +24,7 @@ dayjs.extend(relativeTime)
 dayjs.extend(durationTime)
 
 describe('SelfBidList', () => {
-  test('should display texts', () => {
+  test.skip('should display texts', () => {
```

Comment:
Why are we skipping this test?

Reply:
I wanted to leave refactoring the tests to a later PR.

Reply:
Sure, just make sure not to forget ;)
I am very good at "postponing" writing tests for react